### PR TITLE
GRANT: TOTAL amount when accepting

### PR DESCRIPTION
**Pull request by dogyun-slayte**
Add help article 

* ticket: https://app.asana.com/0/749499898308135/1113491270166781
* Related PRs:
https://github.com/slayte/web/pull/1034
https://github.com/slayte/machine/pull/386
https://github.com/slayte/enum/pull/181
https://github.com/slayte/app/pull/755

### CODE REVIEW

File: assets/inapp.json [hunk 1]
```diff
@@ -262,5 +262,9 @@
   "SubscribeCalendar": {
     "name": "How to subscribe to your appointments ",
     "description": "On this tab you can additionally export your appointment data into an external calendar e.g. Google/Outlook Calendar or even on your phone. To do this:\n\n\nOn the top-left corner of this page click Subscribe Calendar.\nCopy the URL link displayed.\nIn an external calendar of your choice (e.g. Google Calendar) find the option to Add a New Calendar using URL.\nPaste your copied URL from Slayte into this space provided to paste a new calendar URL.\nSave changes."
+  },
+  "Grant_UseTotalsInPayment": {
```
Comment (maximilianvp) [inline assets/inapp.json]: We should rename this into `Specify each payment`.

Change the text like this:
```
By default, Slayte creates and calculates the amount of each payment based on duration and amount which you enter when accepting an awardee.\n
If you'd like more control, select "Specify each payment" -  Then you create exactly the template with payment-events for an awardee, and when accepting the awardee we compute the value of each payment event created by you (also based on sponsoring amount and duration). You may add template payment events with percentage- or total-amount which will turn into those calculated payment-events on the awardee profile when accepted.
```

Comment (dogyun-slayte) [reply assets/inapp.json]: refactored it

File: assets/inapp.json [hunk 2]
```diff
@@ -262,5 +262,9 @@
...
     "description": "On this tab you can additionally export your appointment data into an external calendar e.g. Google/Outlook Calendar or even on your phone. To do this:\n\n\nOn the top-left corner of this page click Subscribe Calendar.\nCopy the URL link displayed.\nIn an external calendar of your choice (e.g. Google Calendar) find the option to Add a New Calendar using URL.\nPaste your copied URL from Slayte into this space provided to paste a new calendar URL.\nSave changes."
+  },
+  "Grant_SpecifyEachPayment": {
+    "name": "Specify Each Payment",
+    "description": "By default, Slayte creates and calculates the amount of each payment based on duration and amount which you enter when accepting an awardee.\nIf you'd like more control, select "Specify each payment" -  Then you create exactly the template with payment-events for an awardee, and when accepting the awardee we compute the value of each payment event created by you (also based on sponsoring amount and duration). You may add template payment events with percentage- or total-amount which will turn into those calculated payment-events on the awardee profile when accepted."
```
Comment (maximilianvp) [inline assets/inapp.json]: Well, this isn't valid JSON. You have `"` in `"`. Please fix

Comment (dogyun-slayte) [reply assets/inapp.json]: fixed

**Comment by dogyun-slayte** [comment]
PR is updated

**Pull request merged**